en/docs/install-and-setup/setup/user-stores/managing-users.md

```diff
@@ -37,7 +37,17 @@ An existing admin user can log in to the Micro Integrator server from the CLI to
 
 ### Non-admin users
 
-Users that do not have admin privileges can access the management API, the CLI, and the dashboard to view and monitor integration artifacts and logs.
+Users that do not have admin privileges can access the management API, the CLI, and the dashboard to update the deployed artifacts, view and monitor integration artifacts and logs.
+
+#### Restrict non-admin users from updating the deployed artifacts
+To restrict the non-admin users from updating the deployed artifacts through **management API** and **CLI**, you can add the following configuration to the `<MI_HOME>/conf/deployment.toml` file. (Available from update level:4.2.0.103)
+
+```toml
+[user_access]
+make_non_admin_users_read_only = true
+```
+
+To restrict the non-admin users from updating the deployed artifacts with **MI dashboard**, you can add the same configuration to the `<MI_DASHBOARD_HOME>/conf/deployment.toml` file. (Available from update level:4.2.0.18)
```

Comment:
shall we give the config here. Saying same configuration can be confusing.

Reply:
Updated with d67de7fab40aa6e2691877a7c4b95d656b0fab8e